src/main/java/io/codelex/arrays/practice/Exercise7.java

```diff
@@ -0,0 +1,115 @@
+package io.codelex.arrays.practice;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Exercise7 {
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        char player1 = 'X';
+        char player2 = 'O';
+
+        //Izveidoju laukumu
+        char[][] ticTacToe = new char[3][3];
```

Comment:
piemēram šeit, kur prasās komentārs, varētu izveidot palīgmetodi ar paskaidrojošu nosaukumu, in general komentārus labāk izlaist pēc iespējas, bet tagad jau protams arī tik mācīšanas process, nedaudz savadāk

Reply:
Pārtaisīju laukuma izveides kodu par palīgmetodi, ja tā biji domājis. 
`char[][] ticTacToe = buildTicTacToe();`